report_substitute/views/ir_actions_report.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="ir_actions_report_form_view">
+        <field name="name">ir.actions.report.form (in report_dispatch_base)
```

Comment:
No need to specify the `<field name="name"> ... ` since you are inheriting. Also `report_dispatch_base` is probably some leftover from old code.

Reply:
good catch

Reply:
If you don't specify the name, Odoo will automatically create one.